examples/fluids/advection.h

```diff
@@ -437,6 +458,88 @@ CEED_QFUNCTION(IFunction_Advection)(void *ctx, CeedInt Q,
   return 0;
 }
 
+// *****************************************************************************
+// This QFunction implements consistent outflow and inflow BCs
+//      for 3D advection
+//
+//  Inflow and outflow faces are determined based on sign(dot(wind, normal)):
+//    sign(dot(wind, normal)) > 0 : outflow BCs
+//    sign(dot(wind, normal)) < 0 : inflow BCs
+//
+//  Outflow BCs:
+//    The validity of the weak form of the governing equations is extended
+//    to the outflow and the values obtained from the previous iteration
```

Comment:
I think it would be better if `from the previous iteration` could be updated to match the description given in the user manual

Reply:
Also, I think we need to update the description at the beginning of this file, lines 83-89:
```
//  Boundary Conditions:
//    Mass Density:
//      0.0 flux
//    Momentum Density:
//      0.0
//    Energy Density:
//      0.0 flux
```
with whichever default BCs we have now, and maybe mention the optional ones (not default ones) as well

Reply:
The same comment/description appears on lines 87-93 of `advection2d.h`

Reply:
Oh, it completely slipped my mind to fix these too. Thanks.